core/src/main/java/org/sonarsource/sonarlint/core/container/global/ExtensionInstaller.java

```diff
@@ -105,7 +101,7 @@ private boolean onlySonarSourceSensor(PluginInfo pluginInfo, Object extension) {
       LOG.debug("TypeScript sensor excluded");
       return false;
     }
-    return pluginVersionChecker.getMinimumVersion(pluginInfo.getKey()) != null || isNotSensor(extension);
+    return Language.containsPlugin(pluginInfo.getKey()) || isNotSensor(extension);
```

Comment:
I think it would be even more explicit to name this method `Language.isSupported()`

Reply:
The method was already there so I would prefer to not change it in this PR.